charts/kargo/templates/shared-resources-namespace/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: {{ include "sharedResources.namespace" . | quote }}
```

Comment:
You'll have to make this conditional like we do for the cluster resources namespace. For various reasons, some users may prefer not to have the chart create this namespace.